docs/aws.md

```diff
@@ -552,12 +552,13 @@ This also serves as an example for users who would like to implement their own A
 
 This client factory has the following configurable catalog properties:
 
-| Property                          | Default                                  | Description                                            |
-| --------------------------------- | ---------------------------------------- | ------------------------------------------------------ |
-| client.assume-role.arn            | null, requires user input                | ARN of the role to assume, e.g. arn:aws:iam::123456789:role/myRoleToAssume  |
-| client.assume-role.region         | null, requires user input                | All AWS clients except the STS client will use the given region instead of the default region chain  |
-| client.assume-role.external-id    | null                                     | An optional [external ID](https://docs.aws.amazon.com/IAM/latest/UserGuide/id_roles_create_for-user_externalid.html)  |
-| client.assume-role.timeout-sec    | 1 hour                                   | Timeout of each assume role session. At the end of the timeout, a new set of role session credentials will be fetched through a STS client.  |
+| Property                          | Default                   | Description                                                                                                                                 |
```

Comment:
no need to change all the lines, just need to add one more line for the new config added

Reply:
Thank you for your review. I think the "change all the lines" here is due to the reformat of the markdown table after I add a new line. The actual contents of previously existing lines remain unchanged. In this case, do I need to rollback to the previous format?